BUILD.bazel

```diff
@@ -26,17 +25,6 @@ alias(
     actual = "@rules_rust//tools/rust_analyzer:gen_rust_project",
 )
 
-# Do not use directly, run `:gen_rust_project` instead.
-rust_analyzer(
-    name = "rust_analyzer",
-    targets = [
-        "//sw/host/opentitanlib:opentitanlib",
-        "//sw/host/opentitanlib/opentitantool_derive:opentitantool_derive",
-        "//sw/host/opentitansession:opentitansession",
-        "//sw/host/opentitantool:opentitantool",
-    ],
-)
-
```

Comment:
Should we maybe just comment this out (along with `gen_rust_project`) and a note that it's temporarily broken and that we're updating rules_rust to fix it?

Reply:
Oh sorry I forgot to mention, the `rust_analyzer` rule is [not required anymore](https://github.com/bazelbuild/rules_rust/pull/1405) it seems. It no longer exists in the latest upstream rules_rust, so I deleted it.